test/units/utils/TestUtils.sol

```diff
@@ -152,4 +153,23 @@ library TestUtils {
         });
         fee = layerZeroContract.quoteSend(sendParam, false);
     }
+
+    function emptyConfigParams() internal pure returns (ConfigLib.CommonConfigParams memory) {
```

Comment:
This is not needed, declaring `ConfigLib.CommonConfigParams memory params;` is enough to create a default empty object.